{"uid":"1d0d0794fc76cc76","name":"loading /frontend/packages/internal/catalyst_voices_services/test/src/auth/auth_service_test.dart","historyId":".frontend.packages.internal.catalyst_voices_services.test.src.auth.auth_service:.frontend.packages.internal.catalyst_voices_services.test.src.auth.auth_service#loading /frontend/packages/internal/catalyst_voices_services/test/src/auth/auth_service_test.dart","time":{"start":1746620346000,"stop":1746620352642,"duration":6642},"status":"broken","statusMessage":"6 errors, see stacktrace for details","statusTrace":"Error #1:\n\nFailed to load \"/frontend/packages/internal/catalyst_voices_services/test/src/auth/auth_service_test.dart\":\nCompilation failed for testPath=/frontend/packages/internal/catalyst_voices_services/test/src/auth/auth_service_test.dart: ../catalyst_voices_repositories/lib/src/wallet/wallet_repository.dart:33:10: Error: The method 'apiV1CardanoAssetsStakeAddressGet' isn't defined for the class 'CatGateway'.\n - 'CatGateway' is from 'package:catalyst_voices_repositories/generated/api/cat_gateway.swagger.dart' ('../catalyst_voices_repositories/lib/generated/api/cat_gateway.swagger.dart').\nTry correcting the name to the name of an existing method, or defining a method named 'apiV1CardanoAssetsStakeAddressGet'.\n        .apiV1CardanoAssetsStakeAddressGet(\n         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n.\\n\\n\\nError #2: Error: the Dart compiler exited unexpectedly.\n\nStacktrace:\npackage:flutter_tools/src/base/common.dart 34:3  throwToolExit\npackage:flutter_tools/src/compile.dart 895:11    DefaultResidentCompiler._compile.<fn>\ndart:async/zone.dart 1538:47                     _rootRunUnary\ndart:async/zone.dart 1429:19                     _CustomZone.runUnary\ndart:async/future_impl.dart 932:45               Future._propagateToListeners.handleValueCallback\ndart:async/future_impl.dart 961:13               Future._propagateToListeners\ndart:async/future_impl.dart 846:9                Future._propagateToListeners\ndart:async/future_impl.dart 712:5                Future._completeWithValue\ndart:async/future_impl.dart 792:7                Future._asyncCompleteWithValue.<fn>\ndart:async/zone.dart 1525:13                     _rootRun\ndart:async/zone.dart 1422:19                     _CustomZone.run\ndart:async/zone.dart 1321:7                      _CustomZone.runGuarded\ndart:async/zone.dart 1362:23                     _CustomZone.bindCallbackGuarded.<fn>\ndart:async/schedule_microtask.dart 40:21         _microtaskLoop\ndart:async/schedule_microtask.dart 49:5          _startMicrotaskLoop\ndart:isolate-patch/isolate_patch.dart 128:13     _runPendingImmediateCallback\ndart:isolate-patch/isolate_patch.dart 195:5      _RawReceivePort._handleMessage\n\\n\\n\\nError #3: Error: the Dart compiler exited unexpectedly.\n\nStacktrace:\npackage:flutter_tools/src/base/common.dart 34:3  throwToolExit\npackage:flutter_tools/src/compile.dart 895:11    DefaultResidentCompiler._compile.<fn>\ndart:async/zone.dart 1538:47                     _rootRunUnary\ndart:async/zone.dart 1429:19                     _CustomZone.runUnary\ndart:async/future_impl.dart 932:45               Future._propagateToListeners.handleValueCallback\ndart:async/future_impl.dart 961:13               Future._propagateToListeners\ndart:async/future_impl.dart 846:9                Future._propagateToListeners\ndart:async/future_impl.dart 712:5                Future._completeWithValue\ndart:async/future_impl.dart 792:7                Future._asyncCompleteWithValue.<fn>\ndart:async/zone.dart 1525:13                     _rootRun\ndart:async/zone.dart 1422:19                     _CustomZone.run\ndart:async/zone.dart 1321:7                      _CustomZone.runGuarded\ndart:async/zone.dart 1362:23                     _CustomZone.bindCallbackGuarded.<fn>\ndart:async/schedule_microtask.dart 40:21         _microtaskLoop\ndart:async/schedule_microtask.dart 49:5          _startMicrotaskLoop\ndart:isolate-patch/isolate_patch.dart 128:13     _runPendingImmediateCallback\ndart:isolate-patch/isolate_patch.dart 195:5      _RawReceivePort._handleMessage\n\\n\\n\\nError #4: Error: the Dart compiler exited unexpectedly.\n\nStacktrace:\npackage:flutter_tools/src/base/common.dart 34:3  throwToolExit\npackage:flutter_tools/src/compile.dart 895:11    DefaultResidentCompiler._compile.<fn>\ndart:async/zone.dart 1538:47                     _rootRunUnary\ndart:async/zone.dart 1429:19                     _CustomZone.runUnary\ndart:async/future_impl.dart 932:45               Future._propagateToListeners.handleValueCallback\ndart:async/future_impl.dart 961:13               Future._propagateToListeners\ndart:async/future_impl.dart 846:9                Future._propagateToListeners\ndart:async/future_impl.dart 712:5                Future._completeWithValue\ndart:async/future_impl.dart 792:7                Future._asyncCompleteWithValue.<fn>\ndart:async/zone.dart 1525:13                     _rootRun\ndart:async/zone.dart 1422:19                     _CustomZone.run\ndart:async/zone.dart 1321:7                      _CustomZone.runGuarded\ndart:async/zone.dart 1362:23                     _CustomZone.bindCallbackGuarded.<fn>\ndart:async/schedule_microtask.dart 40:21         _microtaskLoop\ndart:async/schedule_microtask.dart 49:5          _startMicrotaskLoop\ndart:isolate-patch/isolate_patch.dart 128:13     _runPendingImmediateCallback\ndart:isolate-patch/isolate_patch.dart 195:5      _RawReceivePort._handleMessage\n\\n\\n\\nError #5: Error: the Dart compiler exited unexpectedly.\n\nStacktrace:\npackage:flutter_tools/src/base/common.dart 34:3  throwToolExit\npackage:flutter_tools/src/compile.dart 895:11    DefaultResidentCompiler._compile.<fn>\ndart:async/zone.dart 1538:47                     _rootRunUnary\ndart:async/future_impl.dart 932:45               Future._propagateToListeners.handleValueCallback\ndart:async/future_impl.dart 961:13               Future._propagateToListeners\ndart:async/future_impl.dart 846:9                Future._propagateToListeners\ndart:async/future_impl.dart 712:5                Future._completeWithValue\ndart:async/future_impl.dart 792:7                Future._asyncCompleteWithValue.<fn>\ndart:async/zone.dart 1525:13                     _rootRun\ndart:async/zone.dart 1422:19                     _CustomZone.run\ndart:async/zone.dart 1321:7                      _CustomZone.runGuarded\ndart:async/zone.dart 1362:23                     _CustomZone.bindCallbackGuarded.<fn>\ndart:async/schedule_microtask.dart 40:21         _microtaskLoop\ndart:async/schedule_microtask.dart 49:5          _startMicrotaskLoop\ndart:isolate-patch/isolate_patch.dart 128:13     _runPendingImmediateCallback\ndart:isolate-patch/isolate_patch.dart 195:5      _RawReceivePort._handleMessage\n\\n\\n\\nError #6: Error: the Dart compiler exited unexpectedly.\n\nStacktrace:\npackage:flutter_tools/src/base/common.dart 34:3  throwToolExit\npackage:flutter_tools/src/compile.dart 895:11    DefaultResidentCompiler._compile.<fn>\ndart:async/zone.dart 1538:47                     _rootRunUnary\ndart:async/future_impl.dart 932:45               Future._propagateToListeners.handleValueCallback\ndart:async/future_impl.dart 961:13               Future._propagateToListeners\ndart:async/future_impl.dart 846:9                Future._propagateToListeners\ndart:async/future_impl.dart 712:5                Future._completeWithValue\ndart:async/future_impl.dart 792:7                Future._asyncCompleteWithValue.<fn>\ndart:async/zone.dart 1525:13                     _rootRun\ndart:async/zone.dart 1422:19                     _CustomZone.run\ndart:async/zone.dart 1321:7                      _CustomZone.runGuarded\ndart:async/zone.dart 1362:23                     _CustomZone.bindCallbackGuarded.<fn>\ndart:async/schedule_microtask.dart 40:21         _microtaskLoop\ndart:async/schedule_microtask.dart 49:5          _startMicrotaskLoop\ndart:isolate-patch/isolate_patch.dart 128:13     _runPendingImmediateCallback\ndart:isolate-patch/isolate_patch.dart 195:5      _RawReceivePort._handleMessage\n","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},"afterStages":[],"labels":[{"name":"resultFormat","value":"junit"},{"name":"suite","value":".frontend.packages.internal.catalyst_voices_services.test.src.auth.auth_service"},{"name":"testClass","value":".frontend.packages.internal.catalyst_voices_services.test.src.auth.auth_service"},{"name":"package","value":".frontend.packages.internal.catalyst_voices_services.test.src.auth.auth_service"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"1d0d0794fc76cc76.json","parameterValues":[]}